app/skinGui/conf/positions/left_leg_lower.txt

```diff
@@ -0,0 +1,773 @@
+name	lower left leg

```

Comment:
I would go here with underscore-separated names.
The same applies to all files.

Reply:
> I would go here with underscore-separated names. The same applies to all files.

For all other files we used the name of the corresponding link (see https://icub-tech-iit.github.io/documentation/icub_kinematics/icub-model-naming-conventions/icub-model-naming-conventions/#links), so to keep consistency I would go for:
* `l_lower_leg`
* `l_upper_leg`
* `r_lower_leg`
* `r_upper_leg`

Reply:
I have edited the names in the headers according to the convention.
Sorry about the link issues in the comment, I have fixed that, too, they should work now.
I was not aware of the references in GUI conf files, I will try to add them where needed.